src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -655,8 +586,8 @@ private void removeTerritoriesWhereTransportsAreExposed(final Map<Territory, Pro
       alreadyAttackedWithUnits.addAll(attackMap.get(t).getUnits());
       alreadyAttackedWithUnits.addAll(attackMap.get(t).getAmphibAttackMap().keySet());
     }
-    for (final Territory t : moveMap.keySet()) {
-      moveMap.get(t).getMaxUnits().removeAll(alreadyAttackedWithUnits);
+    for (final Territory t : defendMap.keySet()) {
+      defendMap.get(t).getMaxUnits().removeAll(alreadyAttackedWithUnits);
```

Comment:
Yeah, there is lots of this still as the territory manager has all the maps/list of territories/units but hands them out to other classes still. Next step is to try to move lots of these utility type methods that use the maps/lists into territorymanager.